docs/source/overview/output_files.rst

```diff
@@ -0,0 +1,19 @@
+Output Files
+============
+
+KBMoD outputs a range of information about the discovered trajectories. Each filename includes a user defined suffix, allowing user to easily save and compare files from different runs. Below we use SUFFIX to indicate the user-defined suffix.
+
+The main file that most users will want to access is `results_SUFFIX.txt`. This file contains one line for each trajectory with the trajectory information (x pixel start, y pixel start, x velocity, y velocity), the number of observations seen, the estimated flux, and the estimated likelihood.
+
+The full list of output files is:
+
+* all_ps_SUFFIX.txt - All of the postage stamp images for each found trajectory.
+* filtered_likes_SUFFIX.txt - ???The likelihood of the trajectory computed only after some observations are filtered???
```

Comment:
We can remove the ??? at the beginning and ending. I've confirmed that this is the case.